.codecov.yml

```diff
@@ -1,7 +1,13 @@
 coverage:
   status:
-    project: off
-    patch: off
+    project:
+      default:
+        target: auto
+        threshold: 1%
+    patch:
+      default:
+        target: 80%
```

Comment:
Should this be 85%?

Reply:
Sounds reasonable. Diff coverages should have higher targets than overall.

Reply:
Updated.